src/Redmine/Client.php

```diff
@@ -100,30 +70,10 @@ class Client implements ClientInterface
      */
     protected $customHost = null;
```

Comment:
```suggestion
    protected ?string $customHost = null;
```
?

Reply:
We can not guaranty than someone had extend the Client class and put some other values into protected properties like $customHost. Setting a type would create a breaking change.